### DISCUSSION TITLE
Cüzdan adresi arama

### DISCUSSION THREAD

**Discussion by Yusufknk** [IBAX-io/go-ibax#24] [Bug Submission]
![IMG_20220106_223819](https://user-images.githubusercontent.com/97254047/148441207-64e26257-3c2e-43ba-8350-cf7fa5a9f777.jpg)

Ekosistem üyelerinden herhangi birinin cüzdan adresini arama yerinden girdiğimizde sayfadan atmaktadır.

Cüzdan adres:0618-8388-3386-3183-7745

**Comment by HalloKyle** [comment]
> ![IMG_20220106_223819](https://user-images.githubusercontent.com/97254047/148441207-64e26257-3c2e-43ba-8350-cf7fa5a9f777.jpg)
> 
> Ekosistem üyelerinden herhangi birinin cüzdan adresini arama yerinden girdiğimizde sayfadan atmaktadır.
> 
> Cüzdan adres:0618-8388-3386-3183-7745

Sorry we don't have a Turkish support yet could you translate your questions in English

**Reply by Yusufknk** [reply]
Any vault type search tool from ecosystem members kicks it off the page.

**Reply by Yusufknk** [reply]
@BadAssKyle
